sublime-package.json

```diff
@@ -33,11 +33,18 @@
                       "description": "Shortens the text spans of underlined problems to their first related line.",
                       "scope": "resource"
                     },
+                    "useFlatConfig": {
+                      "scope": "resource",
+                      "type": ["boolean", "null"],
+                      "default": null,
+                      "markdownDescription": "Controls whether flat config should be used or not. This setting requires ESLint version 8.57 or later and is interpreted according to the [ESLint Flat Config rollout plan](https://eslint.org/blog/2023/10/flat-config-rollout-plans/). This means:\n\n - *8.57.0 <= ESLint version < 9.x*: setting is honored and defaults to false\n- *9.0.0 <= ESLint version < 10.x*: settings is honored and defaults to true\n- *10.0.0 <= ESLint version*: setting is ignored. Flat configs are the default and can't be turned off."
+                    },
                     "experimental.useFlatConfig": {
```

Comment:
You should remove this one from `LSP-eslint.sublime-settings` now and add/update new/changed options.

Reply:
removed